keyboards/keebio/rorschach/keymaps/tuesdayjohn/keymap.c

```diff
@@ -0,0 +1,296 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+enum rorschach_layers {
+  _COLEMAK,   // Colemak (default layer)
+  _QWERTY,    // Qwerty
+  _NUMBERS,   // Numbers & Symbols
+  _NUMBERS2,  // Numbers & Function Keys (similar to _NUMBERS; used with _NUMBERS for tri-layer access to _ADJUST)
+  _FUNCTION,  // Function
+  _FUNCTION2, // Function 2 (identical as _FUNCTION; used to allow for easier use of space and backspace while using function layer arrows)
+  _NUMPAD,    // Numpad
+  _ADJUST,    // Adjust layer, accessed via tri-layer feature)
+  _ADJUST2    // Second Adjust layer, accessed outside of tri-layer feature)
+};
+
+enum rorschach_keycodes {
+  COLEMAK = SAFE_RANGE,
+  QWERTY
+};
+
+//Tap Dance Declarations
+enum {
+  ADJ = 0,
```

Comment:
```suggestion
  ADJ,
```

This is not really necessary, the enum numbering always starts at 0.